docs/resources/monitor.md

```diff
@@ -78,7 +78,7 @@ We recommend at least 2x the monitor timeframe for metric alerts or 2 minutes fo
 - `renotify_occurrences` (Number) The number of re-notification messages that should be sent on the current status.
 - `renotify_statuses` (Set of String) The types of statuses for which re-notification messages should be sent. Valid values are `alert`, `warn`, `no data`.
 - `require_full_window` (Boolean) A boolean indicating whether this monitor needs a full window of data before it's evaluated. Datadog strongly recommends you set this to `false` for sparse metrics, otherwise some evaluations may be skipped. If there's a custom_schedule set, `require_full_window` must be false and will be ignored. Defaults to `true`.
-- `restricted_roles` (Set of String) A list of unique role identifiers to define which roles are allowed to edit the monitor. Editing a monitor includes any updates to the monitor configuration, monitor deletion, and muting of the monitor for any amount of time. Roles unique identifiers can be pulled from the [Roles API](https://docs.datadoghq.com/api/latest/roles/#list-roles) in the `data.id` field.
+- `restricted_roles` (Set of String, Deprecated) A list of unique role identifiers to define which roles are allowed to edit the monitor. Editing a monitor includes any updates to the monitor configuration, monitor deletion, and muting of the monitor for any amount of time. Roles unique identifiers can be pulled from the [Roles API](https://docs.datadoghq.com/api/latest/roles/#list-roles) in the `data.id` field. If this attribute is not configured, Terraform will ignore any restricted roles changes in this resource. **Deprecated.** The `restricted_roles` argument is deprecated. Use `datadog_restriction_policy` resource to manage monitor permission instead.
```

Comment:
Which attribute are you referring to in "If this attribute is not configured..."

```suggestion
- `restricted_roles` (Set of String, Deprecated) A list of unique role identifiers to define which roles are allowed to edit the monitor. Editing a monitor includes any updates to the monitor configuration, monitor deletion, and muting of the monitor for any amount of time. Roles unique identifiers can be pulled from the [Roles API](https://docs.datadoghq.com/api/latest/roles/#list-roles) in the `data.id` field. If this attribute is not configured, Terraform ignores any restricted roles changes in this resource. **Deprecated.** The `restricted_roles` argument is deprecated. Use `datadog_restriction_policy` resource to manage monitor permissions instead.
```

Reply:
Thanks, updated my suggestion to include that.